docs/source/guides/network_node/network_node.rst

```diff
@@ -4,10 +4,24 @@
 Running a Network Node
 ======================
 
-To be a node on the NuCypher network, operations are divided into two roles “Staker” and “Worker”.
+To be a node on the NuCypher network, operations are divided into two roles “Staker” and “Worker”. The Staker can be
+thought of as a fiduciary administrator that holds NU and collects rewards. The Worker is bonded to a Staker and
+performs cryptographic work on the network on behalf of the Staker.
 
-.. toctree::
-   :maxdepth: 1
+Due to the complexity of running a node on the network, there are a variety of ways to participate depending on your
+comfort level:
 
-   staking_guide
-   ursula_configuration_guide
+* Delegate custody of NU and work to `CoinList <https://coinlist.co/nucypher-worklock>`_.
```

Comment:
@mswilkison is `CoinList` something we want to refer to by name here (and in the image) or should we instead use something like `3rd Party Custody/Delegator`? You can view the built page and image here: https://99317-100446516-gh.circle-artifacts.com/0/docs/build/html/guides/network_node/network_node.html

Reply:
I think in the docs better not to refer to CoinLIst specifically and instead be generic. It looks like there are more options incoming (https://twitter.com/NuCypher/status/1298639471152230402).

Shouldn't this be the other way around?
<img width="425" alt="Screen Shot 2020-08-26 at 3 19 17 PM" src="https://user-images.githubusercontent.com/2382226/91347229-809a5300-e7d1-11ea-8136-f18704d5bb83.png">

Reply:
Ok.

Ah, yes it should be the other way around